test/spec/actions/split.js

```diff
@@ -1674,7 +1676,7 @@ describe('iD.actionSplit', function () {
                     iD.osmRelation({id: 'r', tags: {type: type}, members: [
                         {id: '-', role: 'from', type: 'way'},
                         {id: '~', role: 'to', type: 'way'},
-                        {id: 'd', role: 'via', type: 'node'}
+                        {id: 'd', role: via, type: 'node'}
```

Comment:
When reading the changes, I did not immediately get this. Maybe a different name like `viaKey` would make it more explicit? (Code editors will also help with proper tool tips and highlighting, though)